### PR TITLE
ManageWikiDefaultPermissions: fix description i18n

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

**Comment by RhinosF1** [comment]
Hmm, I'm not sure if it was ever meant to be 2 messages. It looks like the description may have at one point before the i18n got lost intended to change between norights and rights.

@Universal-Omega: do you recall the history?

**Comment by OAuthority** [comment]
One of the messages is to show if the user doesn't have the right to manage default permissions (or I guess, now reset the cache etc.) 

It's probably easier to just do a conditional check whether the user has the rights or not and list is accordingly.

**Comment by redbluegreenhat** [comment]
That message is from WikiForge, and has never existed in ManageWiki before. It was introduced in #438. I'd like to say we can just get rid of it and return managewikidefaultpermissions only.

File: includes/Specials/SpecialManageWikiDefaultPermissions.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ public function canModify() {
 	}
 
 	public function getDescription() {
-		return $this->msg( $this->canModify() ? 'managewikidefaultpermissions' : 'managewikidefaultpermissions-norights' )->text();
+		return $this->msg( $this->canModify() ? 'managewikidefaultpermissions' : 'managewikidefaultpermissions' )->text();
```
Comment (redbluegreenhat) [inline includes/Specials/SpecialManageWikiDefaultPermissions.php:39]:
```suggestion
		return $this->msg( 'managewikidefaultpermissions' )->text();
```

**Review by redbluegreenhat — CHANGES_REQUESTED** [review]

**Comment by redbluegreenhat** [comment]
I get that message now, just like how Special:ManageWiki/extensions is titled "View this wiki's extensions" for users without permissions to edit it, and "Manage this wiki's extensions" for users that do have them, this does the same but for Special:ManageWikiDefaultPermissions. We're going to be keeping it, I'll make another PR adding that message to i18n.

**Pull request closed without merging**